week2/index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">s
+<head>
+  <meta charset="UTF-8">
+  <mata name="description" content="LinkBrary"></mata>
+  <mata property="og:title" content="LinkBrary"></mata>
+  <mata property="og:url" content="https://week1parkjiwon.netlify.app/"></mata>
+  <mata property="og:type" content="website"></mata>
+  <mata property="og:image" content="https://week1parkjiwon.netlify.app/"></mata>
+  <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>LinkBrary</title>
+  <link rel="stylesheet" href="index.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-gov-dynamic-subset.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard-gov.css" />
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-gov.css" />
+</head>
+<body>
+    <nav>
+      <div class="nav-bar"> 
+        <a id="logo" href="/"><img src="img/logo.svg" alt="LinkBrary logo"></a>
+        <a class="use_site" id="signin" href="/signin.html" alt="로그인사이트로 이동하는 버튼">로그인</a>
+      </div>
+    </nav>
+    <header>
+      <h2 class="title">
+        <span class="point">세상의 모든 정보</span>를<span class="space pc_br"></span>
+        쉽게 저장하고<span class="space"></span> 
+        관리해 보세요
```

Comment:
빈 block을 media query로 관리하는 것도 나쁘지 않지만 flex box로 자연스럽게 wrap되도록 하면 view의 break point에 의존하지 않고 구현 가능해보입니다.
```suggestion
          <p style="display: flex; flex-wrap: wrap; justify-content: center;">
            <span style="display: inline-block;">쉽게 저장하고</span>
            <span style="display: inline-block;">관리해 보세요</span>
          </p>
```